src/exchange/information/office-offer/office-offer.controller.js

```diff
@@ -192,45 +148,45 @@ angular
                 });
         }
 
-        updateAccounts (accounts) {
-            this.accounts = accounts;
-
-            if (!_.isEmpty(accounts)) {
-                const alreadyPresentAccounts = this.selectedAccounts.map((account) => account.primaryEmailDisplayName);
-                this.selectedAccounts = this.selectedAccounts.concat(accounts.list.results.filter((account) => !alreadyPresentAccounts.includes(account.primaryEmailDisplayName)));
-            }
-
-            this.services.$scope.accounts = accounts;
-        }
-
-        countNumberOfCheckedAccounts () {
+        countNumberOfCheckedAccounts (item) {
             if (!_.isEmpty(this.accounts)) {
```

Comment:
`_(this.accounts).isArray() && !_(this.accounts).isEmpty()` I always check the type before using `isEmpty` as `isEmpty` would return `true` for numbers.

Reply:
It's not an array but an object. Eventually I can write it like this:
_(this.accounts).isObject()  && !_(this.accounts).isEmpty()